<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcEditor</class>
 <widget class="QWidget" name="ProcEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proc Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="6" colspan="3">
    <widget class="QListWidget" name="actionList">
     <item>
      <property name="text">
       <string>action1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>action2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>action3</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="upButton">
     <property name="text">
      <string>Move Up</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="downButton">
     <property name="text">
      <string>Move Down</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="editButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>132</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>197</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
